core/shared/platform/android/platform_init.c

```diff
@@ -5,6 +5,10 @@
 
 #include "platform_api_vmcore.h"
 #include "platform_api_extension.h"
+#include "bh_log.h"
+
+#define API_NOT_SUPPORT_ERROR(API, LEVEL) LOG_ERROR("%s() is only supported when compile option ANDROID_PLATFORM >= %s.", #API, #LEVEL);
```

Comment:
In WAMR's design, the platform layer is the lowest layer, it should not depend any upper layer, so here we should not include bh_log.h and call LOG_ERROR. Could you please call __android_log_print instead, how about:
```C
#define API_NOT_SUPPORT_ERROR(API, VERSION) __android_log_print("%s() is only supported when __ANDROID_API__ >= %s.", #API, #VERSION)
```